packages/smoke-tests/package.json

```diff
@@ -4,11 +4,18 @@
   "main": "index.js",
   "license": "MIT",
   "scripts": {
-    "test": "yarn faltest faltest/**/*-test.js"
+    "lint:js": "yarn eslint .",
+    "test": "yarn faltest faltest/**/*-test.js --disable-timeouts --timeouts-override 900000"
```

Comment:
--disable-timeouts is only advised as a debugging tool. is --timeouts-override not enough?

Reply:
you right, forgot disable-timeouts was in there